contrib/sbom.cdx.json

```diff
@@ -0,0 +1,45 @@
+{
+  "bomFormat": "CycloneDX",
+  "specVersion": "1.6",
+  "version": 1,
+  "metadata": {
+    "authors": [
+      {
+        "name": "@VCS_SBOM_AUTHORS@"
+      }
+    ]
+  },
+  "components": [
+    {
+      "type": "library",
+      "bom-ref": "pkg:github/fwupd/fwupd@@VCS_TAG@",
+      "cpe": "cpe:2.3:a:fwupd:fwupd:@VCS_TAG@:*:*:*:*:*:*:*",
+      "name": "fwupd",
+      "version": "@VCS_VERSION@",
```

Comment:
doesn't configuration data need to explicitly populate this?

Reply:
Nah, the uswid cli works it out itself now.

Reply:
Oh magik. I guess it does with the other authors field too then.